frontend/src/components/search/components/common/filter-dropdown/filter-dropdown.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'common/types/types';
+import { useState } from 'hooks/hooks';
+import { IconName } from 'common/enums/components';
+import { Icon } from 'components/common/common';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  filter: string;
+  onChangeFilter: (value: string) => void;
+  title: string;
+  options: {
+    text: string;
+    value: string;
+  }[];
+};
+
+const FilterDropdown: FC<Props> = ({ filter, onChangeFilter, title, options }) => {
+  const [toggleDropdown, setToggleDropdown] = useState(false);
+
+  const onHandleToggleDropdown = (): void => setToggleDropdown(!toggleDropdown);
+  const onHandleChangeFilter = (e: React.FormEvent<HTMLInputElement>): void => onChangeFilter(e.currentTarget.value);
+
+  return (
+    <>
```

Comment:
I think better to add a container for this content to make work with styles easier in the future.

I do not see a reason of using react fragment `<> </>` here